Resources/config/doctrine/model/AuthorizationCode.orm.xml

```diff
@@ -12,7 +12,7 @@
         <field name="userIdentifier" type="string" length="128" nullable="true" />
         <field name="scopes" type="oauth2_scope" nullable="true" />
         <field name="revoked" type="boolean" />
-        <field name="nonce" type="string" nullable="true" />
+        <field name="nonce" type="string" nullable="true" unique="true"/>
```

Comment:
did you check if the nonce is used in any other use-case where it should not be unique?